src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BundlesRespondentService.java

```diff
@@ -152,4 +159,80 @@ public void addToBundlesCollection(CaseData caseData) {
             )
         );
     }
+
+    /**
+     * Removes a hearing bundle from the collection.
+     */
+    public void removeHearingBundles(CaseData caseData) {
+        String selectedParty = caseData.getRemoveBundleDropDownSelectedParty();
+
+        if (SELECT_CLAIMANT_HEARING_BUNDLES.equals(selectedParty)) {
+            removeHearingBundles(caseData, caseData.getBundlesClaimantCollection());
+        } else if (SELECT_RESPONDENT_HEARING_BUNDLES.equals(selectedParty)) {
+            removeHearingBundles(caseData, caseData.getBundlesRespondentCollection());
+        }
+    }
+
+    private void removeHearingBundles(CaseData caseData,
+                                      List<GenericTypeItem<HearingBundleType>> bundlesCollection) {
+
+        List<GenericTypeItem<RemovedHearingBundleItem>> removedHearingBundlesCollection =
+                caseData.getRemovedHearingBundlesCollection();
+        if (removedHearingBundlesCollection == null) {
+            removedHearingBundlesCollection = new ArrayList<>();
+            caseData.setRemovedHearingBundlesCollection(removedHearingBundlesCollection);
+        }
+
+        if (CollectionUtils.isEmpty(bundlesCollection)) {
+            return;
+        }
```

Comment:
How about moving this to line 179 so that the method exist at the earliest instance if no item is in the bundle passed in?

Reply:
addressed: [277c708](https://github.com/hmcts/et-ccd-callbacks/pull/1878/commits/277c708f74011e8eae2d518b1bf6f4b68ebaff41)